docs/en/1_exist_data_model.md

```diff
@@ -499,9 +499,6 @@ pipelines = [
 MMDetection also provides out-of-the-box tools for training detection models.
 This section will show how to train _predefined_ models (under [configs](https://github.com/open-mmlab/mmdetection/tree/master/configs)) on standard datasets i.e. COCO.
 
-**Important**: The default learning rate in config files is for 8 GPUs and 2 img/gpu (batch size = 8\*2 = 16).
-According to the [linear scaling rule](https://arxiv.org/abs/1706.02677), you need to set the learning rate proportional to the batch size if you use different GPUs or images per GPU, e.g., `lr=0.01` for 4 GPUs \* 2 imgs/gpu and `lr=0.08` for 16 GPUs \* 4 imgs/gpu.
-
 ### Prepare datasets
```

Comment:
We can save this line because this function is normally disabled.

Reply:
I add some detail when it is disabled on line `524` : 

```text
If you don't want to use it, the default learning rate in config files is based on batch size of 16, you need to calculate the learning rate manually according to the practical training batch size and change the learning rate in specific config file.
```